mava/wrappers/jumanji.py

```diff
@@ -93,6 +59,84 @@ def step(
         return state, timestep
 
 
+class MultiAgentWrapper(Wrapper):
```

Comment:
I really like this wrapper!!

Reply:
Unfortunately this is not going to continue to work. We *need* a wrapper per jumanji env so that we can correctly pack their obs into our `Observation`. 

For example connectors observation looks like this:
```python
class Observation
    grid
    action_mask
    step_count
```

Then many other jumanji envs have different values here. Only lbf and rware have the same obs names that is why this works now, but it won't continue to work, so we need to separate this per env.

Reply:
I back the idea of a wrapper per env in that case.